src/grpcbox_stream.erl

```diff
@@ -405,10 +406,10 @@ handle_info({'EXIT', _, {grpc_error, {Status, Message}}}, State) ->
 handle_info({'EXIT', _, _Other}, State) ->
     end_stream(?GRPC_STATUS_UNKNOWN, <<"process exited without reason">>, State),
     State;
-handle_info(_, State) ->
+handle_info({timeout,_Ref,<<"grpc-timeout">>}, State) ->
+    end_stream(?GRPC_STATUS_DEADLINE_EXCEEDED, <<"Deadline expired">>, State),
     State.
 
```

Comment:
there is no `handle_info(_, State)` anymore...

Reply:
Will put it back